### ISSUE TITLE
Add basic documentation to repo

### ISSUE THREAD

**Issue by WilliamRoyNelson** [monodop/DankDittiesBot#16]
At a minimum, there should be a documented "help" or "usage" command, that allows for further info from the bot directly.

Ideally, each command would be documented online.

**Comment by monodop** [comment]
Yep, I've been putting this off but we do need a good user guide.

- starting and running the docker image
- environment variables you can tweak
- available commands for chat & voice
- building and running the bot from source code
